Add section for enabling gateway access logging with the Telemetry API (#409)
MicrosoftDocs/azure-aks-docs: articles/aks/istio-gateway-api.md

    updated-per-gateway
    ```

## Enable access logs for the Gateway pods

> With managed Istio enabled on the cluster, the `Telemetry` CRDs are installed automatically. You can use these CRDs to configure access logging for gateway pods by creating a `Telemetry` resource.

Reviewer: ```suggestion
`Telemetry` CRDs are installed automatically with the Istio add-on. You can use the `Telemetry API` to configure access logging for `Gateway` pods.
```

> Create the resource in the `aks-istio-system` namespace to enable access logs for all gateways in the mesh, or in a specific namespace to enable access logs only for gateways in that namespace.

Reviewer: ```suggestion
Create the resource in the `aks-istio-system` namespace to enable access logs for all `Gateway` pods in the mesh, or in a specific namespace to enable access logs only for `Gateway` pods in that namespace.
```

> The following example shows a `Telemetry` resource that enables gateway access logging:

Reviewer: ```suggestion
The following example shows a `Telemetry` resource that enables `Gateway` access logging:
```

```yaml
apiVersion: telemetry.istio.io/v1
kind: Telemetry
metadata:
  name: mesh-default
  namespace: aks-istio-system

> spec:

Reviewer: This comment seems accurate - I would just add a note that the manifest enables access logging across the entire mesh (including for sidecars) and not just for Gateways

    updated-per-gateway
    ```

## Enable access logs for the Gateway pods

`Telemetry` CRDs are installed automatically with the Istio add-on. You can use the `Telemetry API` to configure access logging for `Gateway` pods.

Create the resource in the `aks-istio-system` namespace to enable access logs for all `Gateway` pods in the mesh, or in a specific namespace to enable access logs only for `Gateway` pods in that namespace.

The following example shows a `Telemetry` resource that enables `Gateway` access logging:

```yaml
apiVersion: telemetry.istio.io/v1
kind: Telemetry
metadata:
  name: mesh-default
  namespace: aks-istio-system
spec:
  accessLogging:
  - providers:
    - name: envoy

> 

Reviewer: Nit - extra space here?

Author: extra space removed

```

  > [!NOTE]

>   >  The above manifest enables access logging across the entire mesh including `Istio sidecars` and `Gateway`pods. Use selectors to target specific pods. Refer to the [Istio add-on documentation][istio-telemetry] for more details on how to configure access logging with the `Telemetry API`.

Reviewer: ```suggestion
  >  The above manifest enables access logging across the entire mesh, both for `Gateway` pods as well as sidecar proxies injected by Istio. Use selectors to target specific pods. Refer to the [Istio add-on documentation][istio-telemetry] for more details on how to configure access logging with the `Telemetry API`.
```
